mountpoint-s3/src/main.rs

```diff
@@ -507,7 +507,12 @@ fn mount(args: CliArgs) -> anyhow::Result<FuseSession> {
         match autoconfigure::network_throughput(&instance_info) {
             Ok(throughput) => throughput,
             Err(e) => {
-                tracing::warn!("failed to detect network throughput: {:?}", e);
+                tracing::warn!(
+                    "failed to detect network throughput: {:?}. Using {} gbps as throughput.
+                    Use --maximum-throughput-gbps CLI flag to configure a target throughput appropriate for the instance.",
+                    e,
+                    DEFAULT_TARGET_THROUGHPUT
+                    );
```

Comment:
how's this? I add `\` since the current string would add a newline and white-space/indentation to the logs.

```suggestion
                tracing::warn!(
                    "Failed to detect network throughput, using default {DEFAULT_TARGET_THROUGHPUT} Gbps. \
                    Use '--maximum-throughput-gbps' mount option to configure a different value. Detection failed due to: {e:?}"
                );
```

Reply:
Yes. this would be better. In this case I think we can surface the [error](https://github.com/awslabs/mountpoint-s3/blob/e52d68ede985ecdb31f68e8db2beef29b528c8b1/mountpoint-s3-client/src/instance_info.rs#L31) "EC2 instance not retrieved" at warn level. Earlier, I was thinking to change it to `debug!` as we are again logging it here.